src/pudl/output/epacems.py

```diff
@@ -148,5 +148,5 @@ def epacems(
             years=epacems_settings.years,
         ),
     )
-
+    # TODO: correct_epa_eia_plant_id_mapping!
```

Comment:
Could you elaborate on this TODO, please? Should we create an issue?

Reply:
ah no i moved this into the cems processing itself so this should be deleted